packages/web/.prettierignore

```diff
@@ -8,7 +8,10 @@
 *.svg
 *.png
 *.jpg
+*.webp
+
 *.env
+*.env.local
```

Comment:
I think there can be _many_ flavors of env files so you may want to do `*.env.*` if that works (not sure)

See
https://nextjs.org/docs/pages/building-your-application/configuring/environment-variables#environment-variable-load-order